baseDAO.cabal

```
@@ -1,8 +1,10 @@
 cabal-version: 2.2
 
--- This file has been generated from package.yaml by hpack version 0.34.2.
+-- This file has been generated from package.yaml by hpack version 0.33.0.
 --
 -- see: https://github.com/sol/hpack
+--
+-- hash: c404c8c08042bf68ae4b3c88041d0ec3147bb68849336dddb3303ba233bbeccc
```

It seems you are running an older version of stack compared to CI, I think.
Anyway this should fix it:
```suggestion
-- This file has been generated from package.yaml by hpack version 0.34.2.
--
-- see: https://github.com/sol/hpack
```